package.json

```diff
@@ -24,7 +24,8 @@
     "benchmark:clean": "rm -rf ./benchmark",
     "benchmark:full": "npm run benchmark:make-certs && npm run benchmark:run && npm run benchmark:clean",
     "snyk-protect": "snyk protect",
-    "prepare": "npm run snyk-protect"
+    "prepare": "npm run snyk-protect",
+    "benchmark": "ts-node performance-tests/script.ts "
```

Comment:
there is an extra space =)

let's rename the file-name to what it test

Reply:
Done. renamed to "wrap-performance-test.ts"